src/types/meetingDetailCommentsList.ts

```diff
@@ -0,0 +1,24 @@
+export interface APIDateAt {
+  createdAt: number;
+  modifiedAt: number;
+}
+
+export interface APIBookGroupComments {
+  commentId: number;
+  contents: string;
+  bookGroupId: number;
+  parentCommentId: number;
+  userId: number;
+  userProfileImage: string;
+  createdAt: APIDateAt[];
+  modifiedAt: APIDateAt[];
+}
```

Comment:
p2;

```suggestion
export interface APIDateAt {
  createdAt: number;
  modifiedAt: number;
}

export interface APIBookGroupComments extends APIDateAt {
  commentId: number;
  contents: string;
  bookGroupId: number;
  parentCommentId: number;
  userId: number;
  userProfileImage: string;
}
```

APIDateAt 인터페이스를 상속 받으면 코드 중복을 줄일 수 있을 것 같아요!